xds/src/main/java/io/grpc/xds/ClientLoadCounter.java

```diff
@@ -256,16 +259,17 @@ synchronized void addValue(String metricName, double value) {
   }
 
   /**
-   * An {@link XdsClientLoadRecorder} instance records and aggregates client-side load data into an
+   * An {@link ClientLoadRecorder} instance records and aggregates client-side load data into an
    * {@link ClientLoadCounter} object.
    */
   @ThreadSafe
-  static final class XdsClientLoadRecorder extends ClientStreamTracer.Factory {
+  @VisibleForTesting
+  static final class ClientLoadRecorder extends ClientStreamTracer.Factory {
```

Comment:
Sure, I agree with the first one, though a "recorder" sounds more semantically intuitive. For maintainability, I will follow your approach.

For those such as `XdsLoadStatsStore`, I was struggling to be more consistent, as well as establishing a consensus with @dapengzhang0 such that top-level xDS classes should be prefixed with `Xds` and others should not. But you know, things and structures are changing back and forth recently, the "convention" isn't stable yet. Yes, we should definitely have a PR for cleaning up class naming issues later.

Reply:
@voidzcy Maybe there was misunderstanding, but I don't think we had consensus on any prefix of top-level xDS classes. At least `LocalityStore` and `FallbackManager` are not prefixed. I think we only talked about when we transform an envoy protobuf class into our own class, such as `Locality` or `DropOverload`, we can add a prefix to avoid name conflict with envoy protobuf.